src/data_source/storage/sql.rs

```diff
@@ -215,20 +168,50 @@ fn add_custom_migrations(
 }
 
 /// Postgres client config.
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 pub struct Config {
-    client_config: client::Config,
```

Comment:
Removed the resettable client stuff as it was complicated to update with the new design, and will be obsolete anyways once we switch to sqlx, where the transaction pool does its own monitoring and refreshing of connections